docs/samples/client/csharp/oauth/readme.md

```diff
@@ -13,6 +13,8 @@ This readme is intended to demonstrate using the authentication token provider.
 This constructor initializes a new instance of the `SampleTypeSpecClient` class and sets a client pipeline with OAuth2 bearer token authentication using the provided `AuthenticationTokenProvider` and endpoint URI.
 
 ```csharp
+using System.ClientModel;
+using System.ClientModel.Primitives;
```

Comment:
We should also include the `public partial class SampleTypeSpecClient` and mention that we need to customize the client to add the custom constructor.